mautic/mautic
# Option to change Region for Amazon SES #559

## Description

Feature to change region for Amazon SES done based on feature request in 

https://github.com/mautic/mautic/issues/559

## Steps to test

1. Apply the PR.

2. Clear Mautic cache.

3. Go to "Email Settings" in Mautic configuration.

4. Select "Amazon SES" option in "Service to send mail through" drop-down.

5. Apply the new setting "Amazon SES Host" along with the Amazon SES username and password. 

6. Do "Test connection" and "Send test email" to check if the connection is correct.
